shenyu-sync-data-center/shenyu-sync-data-websocket/src/main/java/org/apache/shenyu/plugin/sync/data/websocket/client/ShenyuWebsocketClient.java

```diff
@@ -129,71 +137,118 @@ public boolean connectBlocking() {
         }
         return success;
     }
-
+    
     @Override
     public void onOpen(final ServerHandshake serverHandshake) {
+        LOG.info("websocket connection server[{}] is opened, sending sync msg", this.getURI().toString());
+        send(DataEventTypeEnum.CLUSTER.name());
         if (!alreadySync) {
             send(DataEventTypeEnum.MYSELF.name());
             alreadySync = true;
         }
     }
-
+    
     @Override
     public void onMessage(final String result) {
-        handleResult(result);
+//        LOG.info("onMessage server[{}] result({})", this.getURI().toString(), result);
+        Map<String, Object> jsonToMap = JsonUtils.jsonToMap(result);
+        Object eventType = jsonToMap.get("eventType");
+        if (Objects.equals("CLUSTER", eventType)) {
```

Comment:
not use magic string, you can create constant variable or enum include `CLUSTER`, `STANDALONE`